tests/network/test_network_actors.py

```diff
@@ -113,16 +115,16 @@ def test_treasure_map_with_bad_id_does_not_propagate(idle_policy, ursulas):
     assert False
```

Comment:
Always fail?

Reply:
Yeah, this is skipped for now.  I'll come back around to it in LL4.  It's possible that this is not really needed anymore.